README.md

```diff
@@ -0,0 +1,187 @@
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby 3.3.0
+- Browser (Google Chrome, Mozilla Firefox, Safari or any other browser)
+
+### Setup
+
+Clone this repository to your dist folder:
+
+```
+git clone git@github.com:Al-Amin-Khan-Shakil/mvrva-transport-front-end.git
+cd mvrva-transport-front-end
+```
+
+### Install
+
+Install this project with:
+npm install
+
+Make sure that server is running
+rails s
+
+### Usage
+
+Use this project in the browsers of desktops and mobile devices
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Victor Kanyi**
+
+- GitHub: [@jkanyi-web](https://github.com/jkanyi-web)
+- Twitter: [@V_Kanyi](https://twitter.com/V_Kanyi)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/victor-kanyi)
+
+👤 **Vangelis**
+
+- [@github](https://github.com/vangelif)
+- [@twitter](https://twitter.com/vangfot)
+- [@linkedin](https://www.linkedin.com/in/vangfot/)
+
+👤 **Robert Kanyoro**
+
+- GitHub: [@Kanyorok](https://github.com/Kanyorok)
+- Twitter: [@CarlosMbugua](https://twitter.com/CarlosMbugua)
+- LinkedIn: [Robert Mbugua](https://www.linkedin.com/in/robert-mbugua-kanyoro/)
+
+👤 **Md Mahadi Hasan Rifat**
+
+- GitHub: (https://github.com/Rifat8080)
+- Twitter: (https://twitter.com/mdmahadirifat98)
+- LinkedIn: (https://www.linkedin.com/in/md-mahadi-hasan-rifat-8b7851265/)
+
+👤 **Al Amin Khan Shakil**
+
+- GitHub: [Al Amin Khan Shakil](https://github.com/Al-Amin-Khan-Shakil)
+- Twitter: [Al Amin Khan Shakil](https://twitter.com/AlAminKhan85004)
+- LinkedIn: [Al Amin Khan Shakil](https://www.linkedin.com/in/al-amin-khan-shakil/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add user notifications feature.
+- Add user settings feature.
+- Add user account deletion feature.
+- Add user account activation feature.
+- Add user account deactivation feature.
+- Add user email verification feature.
+- Add user profile picture upload feature.
+- Add user profile picture delete feature.
+- Add more styling to the app.
+- Add payment feature.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Al-Amin-Khan-Shakil/mvrva-transport-front-end/issues).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please let me know and we can improve it further. You are welcome to support this project by giving suggestions for improvement and leaving a star on the project.
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Acknowledgements and suggestions for improvement are always welcome
+
+- https://stackoverflow.com/questions/53510040/carrierwave-argument-error-nil-location-provided-cant-build-uri-for-an-image
```

Comment:
- [ ] In this section of your readme "Acknowledgements", kindly give credit to the author of the design [Murat Korkmaz](https://www.behance.net/muratk) 👍